src/index.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Routes from './Routes';
+import "../node_modules/slick-carousel/slick/slick.css";
+import "../node_modules/slick-carousel/slick/slick-theme.css";
```

Comment:
```suggestion
import "slick-carousel/slick/slick.css";
import "slick-carousel/slick/slick-theme.css";
```
node_modules에 설치된 내용을 불러올 때는 이렇게 적어주시면 도비니다

Reply:
허걱스........고치겠습니다